containers/matomo/config.ini.php

```diff
@@ -151,9 +152,13 @@
 PluginsInstalled[] = "UserId"
 PluginsInstalled[] = "CustomPiwikJs"
 PluginsInstalled[] = "WAIMatomoTheme"
+PluginsInstalled[] = "UsersManagerOnlyAdmin"
 @MATOMO_PLUGINS_INSTALLED@
 
 [WAIMatomoTheme]
 waiUrl = "https://@MATOMO_WAI_URL@"
 
+[UsersManagerOnlyAdmin]
+users_manager_only_admin_enabled = @MATOMO_PLUGIN_USERSMANAGERONLYADMIN_ENABLED@
+
```

Comment:
Questa configurazione dovrebbe essere gestita come per gli altri plugin (es. TrackerHostname o LoginFilterIp) e non scritta direttamente in `config.ini.php`.